pyrubberband/__init__.py

```diff
@@ -3,5 +3,5 @@
 # CREATED:2015-03-02 11:50:46 by Brian McFee <brian.mcfee@nyu.edu>
 '''A python wrapper for rubberband'''
 
-from .pyrb import *
 from .version import version as __version__
+from .pyrb import *
```

Comment:
I had to do this because of the `python 3.6` problem.

Using `version = attr: pyrubberband.version.version` from the setup.cfg tried to read actual script and import packages in the `__init__.py` before installing packages and it caused ModuleNotFoundError in python 3.6.

Putting the version on top solved the problem. Refer to : [stackoverflow disscussion](https://stackoverflow.com/a/71495114/27460890)

This only occurs in `python 3.6`.